bats/tests/k8s/traefik.bats

```diff
@@ -1,3 +1,8 @@
+# shellcheck disable=SC2030,SC2031
```

Comment:
why these checks are disabled only for traefik and not for the ones files?

Reply:
Look at the line directly below; it points to a bug in shellcheck. The bug is triggered by lines like `[[ "$output" ... ]]`, which only exist in this source file.

We don't want to disable these rules at all, but have to in this file until either the bug in shellcheck is fixed, or we have to rework the test not to trigger it. False positives are not acceptable because we'll eventually want to add this to the Github actions lint step.